my-vault/docker-compose.yaml

```
@@ -0,0 +1,15 @@
+   version: "3.9"
+   services:
+     postgres:
+       image: postgres:latest
+       container_name: postgres_db
+       environment:
+         - POSTGRES_USER=admin
+         - POSTGRES_PASSWORD=adminYourPass
+         - POSTGRES_DB=secrets_db
+       ports:
+         - "5432:5432"
+       volumes:
+         - postgres_data:/var/lib/postgresql/data
+   volumes:
+     postgres_data:
```

Should this be indented like this?
```suggestion
 version: "3.9"
services:
  postgres:
    image: postgres:latest
    container_name: postgres_db
    environment:
      - POSTGRES_USER=admin
      - POSTGRES_PASSWORD=adminYourPass
      - POSTGRES_DB=secrets_db
    ports:
      - "5432:5432"
    volumes:
      - postgres_data:/var/lib/postgresql/data
volumes:
  postgres_data:
```